.env.example

```
@@ -0,0 +1,5 @@
+MONGO_URL=mongodb://your_mongo_url
+MONGO_DB=the_name_of_thedb
+MONGO_COLLECTION=my_awesome_data
```

Suggestion, pour coller avec les conditions réelles en prod, dans la mesure du possible:

```suggestion
MONGO_COLLECTION=Features
```

---

J'aime mieux garder ça "secret" même si ça n'est pas vraiment important
D'ailleurs il faudra qu'on discute des droits d'acces à MongoDb :)